docs/index.md

```diff
@@ -19,52 +19,201 @@ Sections:
 
 Screenshot of the Flowsheet Visualizer:
 
-<img src="sample_fv.png" />
+<img src="sample_fv.png" style="width:50vw"/>
 
 The IDAES-IP [Flowsheet Visualizer](#flowsheet-visualizer) (FV) is a graphical user interface that displays complex *flowsheets* (connected components representing a system or sub-system to be optimized) created with the IDAES-IP Python core software. The FV uses web technologies (HTML, CSS, JavaScript) so is cross-platform and has identical functionality on computers running Windows, Mac, and UNIX.
 
+---
+
 **FV Sections:** [Tutorials](#fv-tutorials) | [How-to](#fv-howto) | [Reference](#fv-reference)
 
+---
+
 <span id="fv-tutorials" />
 
-### Tutorials
+### **Tutorials:**
 
-#### Example flowsheet
+#### **Example flowsheet:**
 
 A simple example flowsheet is included and can be viewed with:
 
- `python -m idaes_ui.fv.example`
-
-This will open the system's default web browser to show the flowsheet and its controls.
+```sh
+idaes-ui-example
+```
 
 <span id="fv-howto" />
 
-### How-to
+### **How-to:**
+
+**Visualize your flowsheet**
+
+*Once you have installed this package, within the same conda environment, you can create your model in a Python script and view it by calling the `visualize()` function, which will display your flowsheet model in a browser window.*
+
+**Steps:**
+
+  1. Import dependencies (import dependencies based on your needs)
+  2. Import the visualiz function
+  3. Generate your flowsheet
+  4. Call the visualize function
+
+  ```python
+  # 1. Import dependencies (import dependencies based on your needs).
+  from pyomo.environ import (
+    Constraint,
+    Var,
+    ConcreteModel,
+    Expression,
+    Objective,
+    SolverFactory,
+    TransformationFactory,
+    value,
+  )
+  from pyomo.network import Arc, SequentialDecomposition
+  from idaes.core import FlowsheetBlock
+  from idaes.models.unit_models import (
+      PressureChanger,
+      Mixer,
+      Separator as Splitter,
+      Heater,
+      StoichiometricReactor,
+  )
+  from idaes.models.unit_models import Flash
+  from idaes.models.unit_models.pressure_changer import ThermodynamicAssumption
+  from idaes.core.util.model_statistics import degrees_of_freedom
+  import idaes.logger as idaeslog
+
+  # 2. Import the visualiz function
+  from idaes_ui.fv import visualize
+
+  # 3. Generate your flowsheet here
+
+  # 4. Call the visualize function
+  # It will open a browser window and run on an available port.
+  visualize(YourFlowsheet, "YourFlowSheetName", loop_forever=True)
+  ```
+  Now you can run `python yourScriptName.py` to view your flowsheet in browser.
 
-#### Start the visualizer
-Assuming your flowsheet object is called `flowsheet`:
-* From within a Jupyter Notebook, `flowsheet.visualize("Flowsheet Name")`
-* From within a Python script, add a keyword argument if you want the visualizer to keep running: `flowsheet.visualize("Flowsheet Name", loop_forever=True)`
+---
 
-#### Save the visualization
+**Save the visualization:**
```

Comment:
This is all going to be out of date soon. I would skip it.

Reply:
Should I proceed by removing this section altogether, or would it be best to leave it in for now and update it when necessary? Please let me know how you would like me to proceed. Thank you!"